content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -162,6 +159,12 @@ The output shows that the PersistentVolumeClaim is bound to your PersistentVolum
     NAME            STATUS    VOLUME           CAPACITY   ACCESSMODES   STORAGECLASS   AGE
     task-pv-claim   Bound     task-pv-volume   10Gi       RWO           manual         30s
 
+Running pod with the duplicate persistent volume.
```

Comment:
I would probably make this as a new heading, just before `Access control`
Also probably rename the title:
```suggestion
# Mounting the same persistentVolume in two places
```

Reply:
ok working on it